### DISCUSSION TITLE
spaCy debug data warning message interpretation

### DISCUSSION THREAD

**Discussion by kanayer** [explosion/spaCy#10647] [Help: Coding & Implementations] [answered]
I have a Thai dataset consisting of `14 663` sentences. It was split in the following way: 

```
- Train: 11 730 sentences (1173 documents)
- Dev: 1 466 (147 documents)
- Test: 1 467 (147 documents)
```

I run `spacy debug data -V` command and got the following warning message: 

<img width="885" alt="Screen Shot 2022-04-12 at 11 42 38 AM" src="https://user-images.githubusercontent.com/22777876/162898953-687436fd-8a7d-4861-a311-6afbab3fc135.png">
 
> 1173 training examples also in evaluation data

 Could you please clarify the meaning of the warning message above? I have checked multiple times and I do not have any duplicates in the dataset, so I was wondering if it implies that train and eval docs are identical? 
 
 Also, if 14 000 sentences are on the lower side of the number of examples, how much would be enough?

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @kannaricci , can you double-check the command you ran? It's possible that the paths to the train and dev datasets are incorrect that's why `debug data` showed that report. One giveaway is that the number of dev documents is different from what you mentioned (147) to what was reported (1173). 

You can configure this in the command-line by passing values to the `--paths.train` and `--paths.dev` parameters, or by updating the config file directly.

**Reply by kanayer** [reply]
Oh, you're right! I had a train path specified for dev path. What a silly mistake! Thanks and sorry for taking your time!
